plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestExasolConnectorTest.java

```diff
@@ -61,25 +61,25 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         return switch (connectorBehavior) {
             // Tests requires write access which is not implemented
             case SUPPORTS_AGGREGATION_PUSHDOWN,
-                 SUPPORTS_JOIN_PUSHDOWN,
-                 SUPPORTS_LIMIT_PUSHDOWN,
-                 SUPPORTS_TOPN_PUSHDOWN -> false;
+                    SUPPORTS_JOIN_PUSHDOWN,
```

Comment:
Please separate a commit. The indentation change is unrelated to timestamp support.